addons/web/static/tests/helpers/utils.js

```diff
@@ -199,6 +199,7 @@ export function triggerHotkey(hotkey, addOverlayModParts = false, eventAttrs = {
     }
 
     window.dispatchEvent(new KeyboardEvent("keydown", eventAttrs));
+    window.dispatchEvent(new KeyboardEvent("keyup", eventAttrs));
```

Comment:
Oh I see :-)
Love it. This is symmetrical !
No other test needs to get adapted with this ?